Gemfile.lock

```diff
@@ -141,12 +141,7 @@ GEM
     formatador (0.2.5)
     globalid (0.4.1)
       activesupport (>= 4.2.0)
-    google-cloud-core (1.2.7)
-      google-cloud-env (~> 1.0)
-    google-cloud-env (1.0.5)
-      faraday (~> 0.11)
-    google-cloud-vision (0.31.0)
-      google-cloud-core (~> 1.2)
+    google-cloud-vision (0.32.2)
```

Comment:
@zaksoup I've tried to ensure the changes to this file are correct and don't include any local modifications that shouldn't be committed. Would be grateful for a quick review!

Reply:
@demobox the `Gemfile.lock` file should never be edited by hand. If you change a version of a gem in the `Gemfile` file and run `bundle update [gem name]` just commit whatever changes are made automatically to `Gemfile.lock` so that everybody else has the same version of the dependency tree.

I assume this is what you did, but if you can just confirm that'd be great!

Reply:
> If you change a version of a gem in the Gemfile file and run bundle update [gem name] just commit whatever changes are made automatically to Gemfile.lock so that everybody else has the same version of the dependency tree

That's basically what I did - the challenge is that I have local changes to my `Gemfile` (specifically, to refer to my local copy of the `rap_sheet_parser`) that I did **not** want to commit.

The detailed version of what I did is:
1. Install new bundle using `bundle install` (will use `bundle update ...` in future)
1. `git add -i Gemfile.lock` to start an interactive add of the changes
1. Added only the changes related to the new bundle, ignoring the ones related to my local `Gemfile` changes

Does that sound OK?